com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitGUI.cs

```diff
@@ -283,7 +284,13 @@ public static void SetupMainTexForAlphaTestGI(this Material material, string col
 
         static public void SetupBaseUnlitPass(this Material material)
         {
-            if (material.HasProperty(kDistortionEnable))
+            if (material.shader.IsShaderGraph())
```

Comment:
put that if before in case someone creates a shadergaph with a property names _DIstortionEnable